CHANGELOG.md

```diff
@@ -90,6 +90,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Added `sendfile64` (#[1439](https://github.com/nix-rust/nix/pull/1439))
 - Added `MS_LAZYTIME` to `MsFlags`
   (#[1437](https://github.com/nix-rust/nix/pull/1437))
+- Added `ptrace::interrupt` method for platforms that support `PTRACE_INTERRUPT`
```

Comment:
Shoot.  I just noticed that your CHANGELOG entry got moved into the wrong part of the file when you rebased.  You'll have to move it up to the top.

Reply:
Whoops, I forgot to stop bors when I found the CHANGELOG.  problem.  Don't worry; I'll take care of it.

Reply:
Oops, sorry about that. Did not notice when rebasing with master. Thanks for fixing!